modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/ObjectRelationship/ObjectRelationship.tsx

```diff
@@ -56,8 +56,12 @@ function getLabel<T extends ObjectMap<any>>(
 
 		return value ? String(value) : '';
 	}
+
+	const localizedValue = value as LocalizedValue<string>;
+
 	const label =
-		(value as LocalizedValue<string>)[defaultLanguageId] ??
+		localizedValue[Liferay.ThemeDisplay.getLanguageId()] ??
+		localizedValue[defaultLanguageId] ??
```

Comment:
Hey @devpedrotavares, what do think about use `getLocalizableLabel` from `object-js-components-web`?

To do that we need to call `object-js-components-web` in `packege.json`

Reply:
Done!